index.bs

```diff
@@ -5872,6 +5872,19 @@ The attributes above are structured within this certificate as such:
       2A                                 -- Firmware version: 42
 ~~~
 
+### Certificate Requirements for Enterprise Packed Attestation Statements ### {#sctn-enterprise-packed-attestation-cert-requirements}
+
+There are two potential sources for enterprise attestations in Packed format.
+
+1. Attestation statements from a hardware device, which was manufactured with the statement and a corresponding private key.
+2. Attestation statements which have been provisioned, such as a platform authenticator configured via managed policy.
+
+For attestation statements provided at manufacturing, the Extension OID `1.3.6.1.4.1.45724.1.1.2` (`id-fido-gen-ce-sernum`)
+MUST be present, containing a unique serial number for the device as an OCTET STRING. The extension MUST NOT be marked as critical.
```

Comment:
Is there any way the RP can detect whether this is the case, and thus enforce this requirement? If not, the only actually actionable requirement in this section is that when the extension is present, the RP can verify that it is not marked critical. It's already a bit awkward that many of the requirements set in §8.2.1 (in particular the requirements for the subject field, other than Subject-OU) can only really be verified by a certification auditor, and not by a WebAuthn RP.

If this requirement is not really actionable for RPs, perhaps we should rephrase this section from a set of requirements to verify, to just a description of how these values can be retrieved and parsed if present?